WooCommerce/Classes/ViewRelated/Dashboard/Settings/POS/POSTabEligibilityChecker.swift

```diff
@@ -45,10 +42,29 @@ protocol POSEntryPointEligibilityCheckerProtocol {
     func checkEligibility() async -> POSEligibilityState
 }
 
-final class POSTabEligibilityChecker: POSEntryPointEligibilityCheckerProtocol {
-    private var siteSettingsEligibility: POSEligibilityState?
-    private var featureFlagEligibility: POSEligibilityState?
+/// Legacy enum containing POS invisible reasons + POSIneligibleReason cases for i1.
+private enum LegacyPOSIneligibleReason: Equatable {
```

Comment:
Good choice making the duplicate the legacy, for source control history purposes.